front/app/containers/Admin/reporting/components/ReportBuilder/Widgets/ChartWidgets/PostsByTimeWidget/PostsByTimeCard.tsx

```diff
@@ -32,7 +32,7 @@ const PostsByTime = ({
 
   return (
     <Box
-      id="e2e-posts-by-time-widget"
+      className="e2e-posts-by-time-widget"
```

Comment:
Why? :)

Reply:
The convention in HTML is to only use `id` for elements that occur once. If they can occur more than once, you should use a `class`. Technically most browsers don't really seem to care, but still good to respect the convention I guess

https://stackoverflow.com/a/544052/7237112